packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -88,8 +88,11 @@ export default class GoogleCalendarService implements Calendar {
           {
             ...calEventRaw.organizer,
             id: String(calEventRaw.organizer.id),
-            organizer: true,
             responseStatus: "accepted",
+            organizer: true,
+            email: calEventRaw.destinationCalendar?.externalId
+              ? calEventRaw.destinationCalendar.externalId
+              : calEventRaw.organizer.email,
```

Comment:
the event was created twice because google adds the event to all attendees email primary calendar. so if a user has a secondary calendar as destination calendar he would see event created on both primary and secondary calendar. we can avoid adding event to primary calendar by simply not passing email here